src/python/WMCore/Services/WMStatsServer/WMStatsServer.py

```diff
@@ -132,3 +132,16 @@ def getProtectedLFNs(self):
         """
         callname = 'protectedlfns'
         return self._getResult(callname, verb="GET")
+
+    def getProtectedLFNsT0(self):
```

Comment:
I could be wrong here, but I think this method is not going to be needed. We will use the same `getProtectedLFNs` method, but this object will be pointing to a different endpoint.

Reply:
Well, I did foresee such an approach (eventually). But until we resolve issue https://github.com/dmwm/WMCore/issues/10874 we cannot tell for sure. Because it may turn out the only way to collect what we need from T0 to be trough a set of queries to WMBS, which may change the method signature...etc. Until then I need the separate method, so I can clearly signal `NotImplementedError`. For the time being I did not want to touch the original method for doing so. Once we know how to properly fetch the list from T0 we  can easelly merge those two methods into one. But I'd leave that to be resolved in the other GH issue, rather than here.